ui/App.tsx

```diff
@@ -143,7 +163,8 @@ const App = () => {
             path={V2Routes.ImagePolicyDetails}
             component={withSearchParams(ImagePolicyDetails)}
           />
-          <Route path={V2Routes.FluxRuntime} component={FluxRuntime} />
+          // TODO should be configurable based on feature flag
```

Comment:
Changing comment to a TSX-compatible one:
```suggestion
          {/* TODO should be configurable based on feature flag */}
```